### DISCUSSION TITLE
请教一下如何在时间胶囊节目添加网站链接

### DISCUSSION THREAD

**Discussion by ghost** [imsyy/home#127] [Q&A] [answered]
一个是目前的 一个是以前的 希望可以实现
![Quicker_20230508_162005](https://user-images.githubusercontent.com/101842679/236773582-85c1a135-131b-480d-ba68-dea30116cf05.png)
![Quicker_20230508_161905](https://user-images.githubusercontent.com/101842679/236773605-5b00c0d0-356f-4f67-8c87-336af87d747f.png)

**Comment by imsyy** [comment]
可以自行更改这个文件： `src/views/Box/index.vue`，目前我还没想好往这里面写什么 😂

**Reply by ghost** [reply]
可以复制主界面那个链接不

**Reply by imsyy** [reply, marked as answer]
> 可以复制主界面那个链接不

可以的，可以把那个组件 import 到这里就行

**Reply by ghost** [reply]
root@ubuntu:/www/wwwroot/home# yarn build
yarn run v1.22.19
$ vite build
vite v3.1.8 building for production...                                                                               18:27:05
✓ 475 modules transformed.                                                                                           18:27:09

 ERROR  [vite-plugin-pwa:build] Single file component can contain only one <template> element                        18:27:09
file: /www/wwwroot/home/src/components/TimeCapsule/index.vue


 ERROR  error during build:                                                                                          18:27:09
SyntaxError: Single file component can contain only one <template> element
    at createDuplicateBlockError (/www/wwwroot/home/node_modules/@vue/compiler-sfc/dist/compiler-sfc.cjs.js:5179:17)
    at /www/wwwroot/home/node_modules/@vue/compiler-sfc/dist/compiler-sfc.cjs.js:5114:33
    at Array.forEach (<anonymous>)
    at Object.parse (/www/wwwroot/home/node_modules/@vue/compiler-sfc/dist/compiler-sfc.cjs.js:5087:18)
    at createDescriptor (file:///www/wwwroot/home/node_modules/@vitejs/plugin-vue/dist/index.mjs:62:43)
    at transformMain (file:///www/wwwroot/home/node_modules/@vitejs/plugin-vue/dist/index.mjs:2158:34)
    at Object.transform (file:///www/wwwroot/home/node_modules/@vitejs/plugin-vue/dist/index.mjs:2646:16)
    at file:///www/wwwroot/home/node_modules/vite/node_modules/rollup/dist/es/shared/rollup.js:22710:40

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.


不知道咋改 可以说一下吗
